action.yml

```diff
@@ -30,9 +30,9 @@ runs:
           while IFS= read -r line; do
             if [[ $line != \#* ]]; then
               NAME="$(echo $line | cut -d' ' -f1)"
-              if [ "${{inputs.name_position}}" == "prefix" ]; then
+              if [ -n "${{inputs.prefix}}" ]; then
```

Comment:
this doesn't prevent someone from adding both a `prefix` and a `postfix`, which seems ok?